requirements.txt

```diff
@@ -1,6 +1,6 @@
 joblib>=0.11
 Cython>=0.29,!=0.29.18
-numpy>=1.19.3
+numpy>=1.21
```

Comment:
Do we also need to update `build_tools/build_requirements.txt`?

Reply:
Good catch. I don't remember why we even have those separated tbh